pyproject.toml

```diff
@@ -14,20 +17,26 @@ log_cli_level = "INFO"
 line-length = 99
 target-version = ["py38"]
 
-[tool.isort]
-line_length = 99
-profile = "black"
-
 # Linting tools configuration
-[tool.flake8]
-max-line-length = 99
-max-doc-length = 99
+[tool.ruff]
+line-length = 99
```

Comment:
Our other projects use a line length of 110. It would be good to use that value here for consistency as well.

Reply:
I am actually in favor of using `line-length = 99`. Some of our in-house projects break [python style guide suggestions on maximum line length](https://peps.python.org/pep-0008/#maximum-line-length), and I'm in favor of staying within the lines here.

Reply:
OK.